backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4023,4 +4023,96 @@ databaseChangeLog:
+                    nullable: false
+              - column:
+                  name: name
+                  type: text
+                  remarks: The name of the flag. It should describe the feature it represents.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: value
+                  type: boolean
+                  remarks: The value of the flag. It indicates if the feature is enable or disable for the app.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_at
+                  type: DATETIME
+                  remarks: The creation timestamp for this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: created_by
+                  type: uuid
+                  remarks: The user who created this entity.
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk__anyentity__created_by
+                    references: api_user
+              - column:
+                  name: updated_at
+                  type: DATETIME
+                  remarks: The timestamp for the most recent update of this entity.
+                  constraints:
+                    nullable: false
+              - column:
+                  name: updated_by
+                  type: uuid
+                  remarks: The user who most recently updated this entity.
+                  constraints:
+                    nullable: false
+                    references: api_user
+                    foreignKeyName: fk__anyentity__updated_by
+        - sql: |
+            GRANT SELECT ON ${database.defaultSchemaName}.feature_flag TO ${noPhiUsername};
+      rollback:
+        dropTable:
+          tableName: feature_flag
+  - changeSet:
+      id: insert-multiplex-flag
+      author: johanna@skylight.digital
+      comment: inserts multiplex flag in table
+      changes:
+        - tagDatabase:
+            tag: insert-multiplex-flag
+        - sql:
+            remarks: Insert multiplexEnabled flag with value equal true.
+            sql: |
+              WITH migration_user as (
+                 SELECT internal_id as migration_user_id
+                 FROM ${database.defaultSchemaName}.api_user
+                 WHERE login_email = '${migrations_user_email}'
+               )
+              INSERT INTO ${database.defaultSchemaName}.feature_flag (internal_id, name, value,
+              created_at, created_by, updated_at, updated_by)
+              SELECT
+                gen_random_uuid(),
+                'multiplexEnabled',
+                true,
+                now(),
+                migration_user_id,
+                now(),
+                migration_user_id
+              FROM migration_user
+      rollback:
+        truncateTable:
+          tableName: feature_flag
```

Comment:
@emmastephenson I want to confirm if this is the rollback we do for insertions. Or if this simple insertion should have a dummy rollback like the SELECT 'foo'.

Reply:
In this case truncating the table is the right move. The dummy rollback you see above was because we were doing a data migration that would be difficult to undo, but here reverting the table back to an empty state seems appropriate.